src/connections/destinations/catalog/loops/index.md

```diff
@@ -0,0 +1,70 @@
+---
+# The end name should be similar to `Slack  Destination`
+title: Loops
+hide-boilerplate: true
+hide-dossier: true
+---
+
+{% include content/plan-grid.md name="actions" %}
```

Comment:
@danrowden what does this line do

Reply:
It adds this box into the page.

![Screenshot 2023-09-29 at 08 21 29](https://github.com/Loops-so/segment-docs/assets/935215/756be08c-2177-42a0-aa74-7e9176541221)

Here's the template I worked from: https://github.com/segmentio/action-destinations/blob/main/docs/doc-template-new.md?plain=1

Reply:
I'm not actually sure. It's in [the template that Segment provide](https://github.com/segmentio/action-destinations/blob/main/docs/doc-template-new.md?plain=1) and in a lot of pages in the Destinations section of the docs, but doesn't seem to ever render on the web.

Here's an example: https://segment.com/docs/connections/destinations/catalog/actions-canny/